src/map/status.cpp

```diff
@@ -1065,6 +1065,36 @@ void initChangeTables(void)
 
 	set_sc( WE_CHEERUP				, SC_CHEERUP		, EFST_CHEERUP		, SCB_STR|SCB_AGI|SCB_VIT|SCB_INT|SCB_DEX|SCB_LUK );
 
+	// Star Emporer
```

Comment:
Correct name "Emperor" xD

Reply:
This is just a comment, at the time I didn't have time to continue the implementation